### DISCUSSION TITLE
Triggering snippets by executing them

### DISCUSSION THREAD

**Discussion by danielo515** [johnlindquist/kit#1091] [Q&A]
Hello.
I was wondering if it is possible to trigger snippets just executing them?
I tried going to snippets, then selecting one and executing but nothing happened. But to be fair, it was also not working with the normal invocation, so it may be related to keyboard typing.

**Comment by johnlindquist** [comment]
@danielo515 My guess is you need to 

```
await hide()
```

before a keyboard type. Otherwise it's sending keystrokes to the open prompt.

If that's not it, can you share your script?

**Reply by danielo515** [reply]
That worked like a charm. I added it to my snippet template. Maybe worth adding it to the default template too
